eslint-plugin-prettier.js

```diff
@@ -160,11 +161,11 @@ const eslintPluginPrettier = {
         const source = sourceCode.text;
 
         return {
-          Program() {
+          Program(node) {
             if (!prettierFormat) {
               // Prettier is expensive to load, so only load it if needed.
-              prettierFormat = require('synckit').createSyncFn(
-                require.resolve('./worker'),
+              prettierFormat = /** @type {PrettierFormat} */ (
```

Comment:
Is the casting here required? It should work fine without it, shouldn't?

Reply:
Will get an TS error `Type '(...args: any[]) => unknown' is not assignable to type '(source: string, options: Options, fileInfoOptions: FileInfoOptions) => string'.
  Type 'unknown' is not assignable to type 'string'.` if no type casting.